### DISCUSSION TITLE
disable logs?

### DISCUSSION THREAD

**Discussion by ninmba** [evilsocket/opensnitch#846] [Q&A] [answered]
Hello, 
Is it possible to disable logs like dns resolve, or disable logs in Addresses tab, Ports tab too? 
thanks.

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
Hi @ninmba ,

What version are you using? There's an option on >= v1.6.0

You can exclude events that matched a rule from being logged to the Events view. Edit the rule you want to exclude events from, and select `[x] Don't log connections`

**Comment by ninmba** [comment]
you are right, i'm using 1.5
i will upgrade it.
thank you!
